packages/gitbook/src/components/Header/HeaderMobileMenu.tsx

```diff
@@ -39,6 +39,7 @@ export function HeaderMobileMenu(props: Partial<React.ButtonHTMLAttributes<HTMLB
     // Close the navigation when navigating to a page
     useEffect(() => {
         document.body.classList.remove(globalClassName);
+        window.scrollTo(0, 0);
```

Comment:
Should we put this somewhere else?

It's confusing at the moment as it applies on all devices but because of the name of the component, it seems tied to mobile.

Reply:
That is confusing. I've combined it with `useScrollToHash` as a `useScrollPage` hook that's runs when pathname or hash changes.